spec/dummy/db/schema.rb

```diff
@@ -11,32 +11,39 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20140610093243) do
+ActiveRecord::Schema.define(version: 20150617152432) do
 
-  create_table "document_numbers", force: true do |t|
+  create_table "document_numbers", force: :cascade do |t|
     t.string   "document"
-    t.integer  "number",     null: false, default: 1
+    t.integer  "number",     default: 1, null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.